components/RevisionCard.vue

```diff
@@ -175,6 +175,62 @@
           <span class="sr-only">{{$t(`Label-Loading`)}}...</span>
         </div>
       </div>
+      <div v-if="enableCesp">
+        <div v-if="displayChoiceAuthor && (choiceInfoAuthor.type == 'warning')" v-bind:style="{color: 'red', 'margin-left': '50px', 'margin-right': '50px'}">
+          WikiLoop-DoubleCheck has detected suspicious behavior by the author of this revision. Recent revisions by this author have an average ORES damaging score of {{choiceInfoAuthor.percentage}}%. Should a warning be sent on your behalf to the author? <br>
+          Previous revisions by this author:
+        </div>
+        <div v-if="displayChoiceAuthor && (choiceInfoAuthor.type == 'block')" v-bind:style="{color: 'red', 'margin-left': '50px', 'margin-right': '50px'}">
+          WikiLoop-DoubleCheck has detected suspicious behavior by the author of this revision. Recent revisions by this author have an average ORES damaging score of {{choiceInfoAuthor.percentage}}%. This author has been warned {{warningThresholdAuthor}} times in the past {{warningTimeframeAuthor}} days. Should a block request be sent on your behalf to the community administrators? <br> 
+          Previous revisions by this author: 
+        </div>
+        <li v-if= "displayHistoryAuthor" v-for="item in previousRevisionInfosAuthor":key="item.timestamp" v-bind:style="{'margin-left': '65px', 'margin-right': '50px'}">
+          <a v-bind:href= "'https://en.wikipedia.org/w/index.php?title='+ item.title + '&diff=prev&oldid=' + item.parentid"> Revision on article {{item.title}}, at {{item.timestamp}}, ORES damaging score: {{item.score}}% </a>
```

Comment:
Instead of `'https://en.wikipedia.org/w/index.php?title='+ item.title + '&diff=prev&oldid=' + item.parentid`

You could write something like `\`https://en.wikipedia.org/w/index.php?title=${item.title}&diff=prev&oldid=${item.parentid}\``